src/visualization/shaders/trigram/vshader.glsl

```diff
@@ -19,7 +19,9 @@
 uniform mat4 xfrm;
 uniform usamplerBuffer tx;
 uniform uint sz;
-uniform float c_cyl, c_sph, c_pos;
+uniform float tz;
+uniform float c_cyl, c_sph, c_pos, c_ort;
+uniform float ptsz_factor;
```

Comment:
`point_size_factor` would be better IMO, it's easier to read and it's used only in two places, so the lengthiness is not annoying too much.